config/jobs/kubernetes/sig-arch/conformance-audit.yaml

```diff
@@ -1,3 +1,54 @@
+presubmits:
+  kubernetes/kubernetes:
+    # same as ci-audit-kind-conformance, but a presubmit job
+    - name: pull-audit-kind-conformance
+      cluster: k8s-infra-prow-build
+      optional: true
+      always_run: false
+      decorate: true
+      skip_branches:
+        - release-\d+\.\d+ # per-release settings
+      labels:
+        preset-service-account: "true"
+        preset-dind-enabled: "true"
+        preset-kind-volume-mounts: "true"
+      decoration_config:
+        timeout: 150m
+        grace_period: 15m
+      path_alias: k8s.io/kubernetes
+      spec:
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/krte:v20250815-171060767f-master
+            command:
+              - wrapper.sh
+              - bash
+              - -c
+              -
+                  curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/"
+                  && curl -sO https://raw.githubusercontent.com/ii/kind/ci-audit-logging/hack/ci/e2e-k8s.sh
```

Comment:
A comment with the link to non-raw file would be nice: https://github.com/ii/kind/blob/ci-audit-logging/hack/ci/e2e-k8s.sh

Are we okay with Kubernetes CI pulling from a private fork without locking down to a specific hash?

Reply:
@pohly i copied what was already in the periodic job. we have to revisit both.